acj/api/ltilaunch.py

```diff
@@ -91,6 +98,48 @@ def status():
         "course_role": String #the SystemRole string value for the current_user
     }
 }
-    todo: query db for data to return, assemble into JSON object, return obj
     """
-    return "some JSON"#jsonify({"error": 'Invalid login data format. Expecting json.'}), 400
+    isLTISession = False
+    if sess.get('LTI'):
+        isLTISession = True
+    
+    body = {}
+    body['valid'] = isLTISession
+
+    assignment = {}
+    assignment['id'] = ""
```

Comment:
NIT: I'd prefer the ids set to None by default to return null instead of empty string